cardano-node/app/cardano-cli.hs

```diff
@@ -60,17 +67,20 @@ opts =
          \ pretty-printing..) for different system generations."
     )
 
-data CLI
-  = CLI
-      { protocol    :: Protocol,
-        mainCommand :: ClientCommand
-        }
+data CLI = CLI
+  { protocol    :: Protocol
+  , mainCommand :: ClientCommand
+  , commonCli   :: CommonCLI
+  , loggingCli  :: LoggingCLIArguments
+  }
```

Comment:
After some thinking, this is a good idea, to avoid repeating the same init across different commands.